src/components/Navigation/Footer.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
```

Comment:
Is it possible to ensure there is padding above the footer so that it is always at the bottom? Or is the assumption that when the pages are implemented they will all be large enough that the footer will be pushed to the bottom

Reply:
So for each of the pages, I'm thinking we're gonna have a css styling where that sets the minimum height of the viewport to be 100% like so:

.page-container {
  position: relative;
  min-height: 100vh;
}